packages/x-codemod/src/v6.0.0/data-grid/preset-safe/index.ts

```diff
@@ -1,10 +1,12 @@
 import transformColumnMenu from '../column-menu-components-rename';
 import removeDisableExtendRowFullWidth from '../remove-disableExtendRowFullWidth-prop';
+import renameRowsPerPageOptions from '../rename-rowsPerPageOptions-prop';
 import { JsCodeShiftAPI, JsCodeShiftFileInfo } from '../../../types';
 
 export default function transformer(file: JsCodeShiftFileInfo, api: JsCodeShiftAPI, options: any) {
   file.source = transformColumnMenu(file, api, options);
   file.source = removeDisableExtendRowFullWidth(file, api, options);
+  file.source = renameRowsPerPageOptions(file, api, options);
```

Comment:
Nitpick: This transformation is done last, but in the docs placed in the middle (second).
Doesn't make a lot of difference, but when reviewing and seeing the first files I would have assumed that the order of transformations would matter for this codemod. 🙈

Reply:
Nice catch 😄 Thanks